pdp/static/js/vic_gen1_app.js

```diff
@@ -72,6 +72,20 @@ $(document).ready(function () {
         }
     );
 
+    // Specify full timeseries download by setting to min/max dates
+    $("#download-full-timeseries").change(
```

Comment:
Since this on-change handler is exactly the same as that in canada_ex_app, and (for better or for worse), it doesn't relies on global knowledge (e.g. the element IDs #from-date and #to-date), you could probably just include it once at the end of `getDateRange()`.